node/src/components/small_network.rs

```diff
@@ -550,6 +556,7 @@ where
         effect_builder: EffectBuilder<REv>,
         peer_id: NodeId,
         transport: Transport,
+        maximum_net_message_size: u32,
```

Comment:
Does it have to be an argument? Can't you take that from `self`?

Reply:
Good catch!